index.html

```diff
@@ -2433,34 +2433,30 @@ <h2>Production and Consumption</h2>
         </li>
       </ol>
 
-      <p class="note" title="Representation-specific entries">
-Note that representation-specific entries will only have special processing
-rules defined by a single <a>representation</a>. Consumers of a different
-<a>representation</a> are required to preserve these entries in the
-<a>DID document</a> <a
-href="#data-model">data model</a> using their generic type processing rules
-to enable lossless conversion of <a>representations</a>. Similarly, producers
-are required to treat entries containing representation-specific syntax using
-generic type processing rules when producing a <a>representation</a> for which
-the entry is not defined. <a>Representations</a> are required to define producer
-behavior for any such entries defined by the <a>representation</a>.
-      </p>
-
       <p>
 The requirements for all <a>conforming producers</a> are as follows:
       </p>
 
       <ol>
         <li>
-A <a>conforming producer</a> MUST serialize all entries in the <a
-href="#data-model">data model</a> that do not have explicit processing rules for
-the <a>representation</a> being produced using only the <a>representation</a>'s
-data type processing rules.
+A <a>conforming producer</a> MUST take a <a>DID document</a> <a
+href="#data-model">data model</a> and a representation-specific entries <a
+data-cite="INFRA#maps">map</a> as input into the <a>production</a> process.
+The <a>conforming producer</a> MAY accept additional options as input
+into the <a>production</a> process.
+        </li>
+        <li>
+A <a>conforming producer</a> MUST serialize all entries in the <a>DID
+document</a> <a href="#data-model">data model</a>, and the
+representation-specific entries <a data-cite="INFRA#maps">map</a>, that do not
+have explicit processing rules for the <a>representation</a> being produced
+using only the <a>representation</a>'s data type processing rules and
+return the serialization after the <a>production</a> process completes.
         </li>
         <li>
-A <a>conforming producer</a> MUST indicate which <a>representation</a> has been
-used for a <a>DID document</a> via a MIME type as described in <a
-href="#did-resolution-metadata"></a>.
+A <a>conforming producer</a> MUST return the MIME type <a
```

Comment:
```suggestion
A <a>conforming producer</a> MUST return the Media Type <a
```
_MIME Types_ became _Media Types_ with [RFC 1590](https://tools.ietf.org/html/rfc1590) in 1994.  (There are several, if not many, places this change is needed.)

Reply:
Yes, I'm going to do this in a big spec-wide PR in one go. Rejecting your change suggestion, even though it's correct, want to bundle all those MIME type -> Media Type changes together /after/ the spec has settled.

Reply:
Fair ... though I don't know what would be lost by doing some of these changes sooner, and think that having a mix of MIME & Media would be more likely to flag the need to replace the former than having all MIME would.  I'll count on you having a list of such "will do these later in batches" changes.

Reply:
Fixed in d915bea303b36c00ec59b27b8ac10845da828fea.